From ab0f7d33c9ba8fe7557dc89bb490bee837652af9 Mon Sep 17 00:00:00 2001
From: Kevron Rees <kevron.m.rees@intel.com>
Date: Tue, 25 Sep 2018 10:12:53 -0700
Subject: [PATCH] add example ias-setup script

Signed-off-by: Kevron Rees <kevron.m.rees@intel.com>
---
 ias-setup | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 ias-setup

diff --git a/ias-setup b/ias-setup
new file mode 100644
index 0000000..2219f23
--- /dev/null
+++ b/ias-setup
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+mkdir -p /run/ias
+chmod 0700 /run/ias
+chown ias:ias /run/ias || echo "may not have ias user yet..."
+
+#user setup if necessary
+id -u ias && exit 0
+
+useradd -u 1099 -G video -d /home/ias ias
+mkdir /home/ias/.config
+chown ias:ias /home/ias/.config
+cp /usr/share/xdg/weston/ias.conf /home/ias/.config/
+
+echo "XDG_RUNTIME_DIR=$XDG_RUNTIME_DIR" >> /home/ias/.bashrc
+
+chown ias:ias /run/ias
-- 
2.17.1

